src/components/Sidebar/NewSidebar.tsx

```diff
@@ -1,3 +1,35 @@
+import { useUser } from "@dashboard/auth";
+import { Box, List, Text } from "@saleor/macaw-ui/next";
 import React from "react";
+import { useIntl } from "react-intl";
+import { Link } from "react-router-dom";
 
-export const NewSidebar = () => <div>Work in progress</div>;
+import useMenuStructure from "./useMenuStructure";
+
+export const NewSidebar = () => {
+  const intl = useIntl();
+  const { user } = useUser();
+  const [menuStructure] = useMenuStructure(intl, user);
+
+  return (
+    <Box
+      backgroundColor="subdued"
+      as="aside"
+      padding={5}
+      borderColor="neutralPlain"
+      borderRightWidth={1}
+      borderRightStyle="solid"
+    >
+      <List as="ol">
+        {menuStructure.map(menuItem => (
+          <Link to={menuItem.url} key={menuItem.id}>
+            <List.Item paddingX={4} paddingY={4} gap={5} borderRadius={3}>
+              {menuItem.icon}
+              <Text>{menuItem.label}</Text>
+            </List.Item>
+          </Link>
```

Comment:
IMO list item should be above link
```suggestion
          <List.Item paddingX={4} paddingY={4} gap={5} borderRadius={3}>
            <Link to={menuItem.url} key={menuItem.id}>
              {menuItem.icon}
              <Text>{menuItem.label}</Text>
            </Link>
          </List.Item>
```

Reply:
Fixed - thank you 🙇🏻